centreon/www/include/options/accessLists/actionsACL/DB-Func.php

```
@@ -174,39 +180,59 @@ function deleteActionInDB($actions = [])
-        foreach ($queryValues as $bindParameter => $bindValue) {
-            $statement->bindValue($bindParameter, $bindValue, PDO::PARAM_INT);
+    $deleteActions = $pearDB->prepare('DELETE FROM acl_actions WHERE acl_action_id = :id');
+    $deleteRules = $pearDB->prepare('DELETE FROM acl_actions_rules WHERE acl_action_rule_id = :id');
+    $deleteRelations = $pearDB->prepare('DELETE FROM acl_group_actions_relations WHERE acl_action_id = :id');
+    $selectNameStmt = $pearDB->prepare(
+        'SELECT acl_action_name FROM acl_actions WHERE acl_action_id = :id LIMIT 1'
+    );
+
+    $pearDB->beginTransaction();
+    try {
+        foreach ($actions as $key => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)